### PR TITLE
Provide documentation for writing dependencies

### PR DESCRIPTION
**Pull request by tomkerkhove**
Relates to #16

### CODE REVIEW

File: docs/features/writing-different-telemetry-types.md [hunk 1]
```diff
@@ -24,6 +24,64 @@ This feature requires to install our NuGet package
...
+
+```csharp
+var telemetryContext = new Dictionary<string, object>
+{
+    { "Catalog", "Products"},
+    { "Tenant", "Contoso"},
+};
+
+// Start measuring
+var startTime = DateTimeOffset.UtcNow;
```
Comment (stijnmoreels) [inline docs/features/writing-different-telemetry-types.md]: With scoped log entries or so, we could maybe have something that does this for us.

File: src/delete-me/delete-me.csproj [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk.Worker">
```
Comment (stijnmoreels) [inline src/delete-me/delete-me.csproj]: Is this on purpose? This extra project?

File: docs/features/writing-different-telemetry-types.md (hunk 1 above)
Comment (tomkerkhove) [reply docs/features/writing-different-telemetry-types.md]: See #46 ;)

File: src/delete-me/delete-me.csproj (hunk 2 above)
Comment (tomkerkhove) [reply src/delete-me/delete-me.csproj]: I see nothing ;)

**Pull request merged**